### PR TITLE
feat: use semantic html and update footer content

### LINKED ISSUE #2047: Some content is not inside a landmark
### Describe the bug:
Best practice is to have all content on the page within a landmark element. While a `header` element is being used for the header content at the top of the page and a `nav` element is being used for the main menu, the main content and footer content are not in landmarks.

### Solution
[ ] Convert `#main-content` into a `main` element.
[ ] Convert the `div` that currently wraps the footer content into a `footer` element.

### PR DESCRIPTION
**Pull request by Sboonny**
Co-authored-by: gikf <60067306+gikf@users.noreply.github.com>
Co-authored-by: Oliver Eyton-Williams <ojeytonwilliams@gmail.com>

- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

Closes #2047

### CODE REVIEW

File: client/src/components/PageLayout/component/Footer.tsx [hunk 1]
```diff
@@ -1,68 +1,33 @@
...
-      <Box
-        display="inline-grid"
-        placeItems={{ base: 'flex-start', lg: 'center' }}
-        width="100%"
+      <Link href="/chapters" fontWeight="600">
+        Chapters
+      </Link>
+      <Link href="/events" fontWeight="600">
+        Events
+      </Link>
```
Comment (ojeytonwilliams) [inline client/src/components/PageLayout/component/Footer.tsx:20]: This is why the test is failing.  It's looking for link with an accessible name `Events`, finding this and clicking on it.

The solution is, I think, to improve the accessibility of the dashboard by using `Events Dashboard` for those links.  I'll put together a quick PR and then this one will start passing.

File: cypress/e2e/dashboard/events/events-index.cy.ts [hunk 2]
```diff
@@ -100,7 +100,6 @@ describe('spec needing owner', () => {
     cy.findByRole('menuitem', { name: 'Dashboard' }).click();
 
     cy.findByRole('link', { name: 'Events' }).click();
-    cy.contains('Loading...');
```
Comment (ojeytonwilliams) [inline cypress/e2e/dashboard/events/events-index.cy.ts:103]: Can you revert this? I'll address the tests in another PR.

File: cypress/e2e/dashboard/events/events-index.cy.ts [hunk 3]
```diff
@@ -140,7 +139,6 @@ describe('spec needing owner', () => {
     cy.get('button[aria-label="Options"]').click();
     cy.findByRole('menuitem', { name: 'Dashboard' }).click();
     cy.findByRole('link', { name: 'Events' }).click();
-    cy.contains('Loading...');
```
Comment (ojeytonwilliams) [inline cypress/e2e/dashboard/events/events-index.cy.ts:143]: This too.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: cypress/e2e/dashboard/events/events-index.cy.ts [hunk 4]
```diff
@@ -100,7 +100,6 @@ describe('spec needing owner', () => {
     cy.findByRole('menuitem', { name: 'Dashboard' }).click();
 
     cy.findByRole('link', { name: 'Events' }).click();
```
Comment (Sboonny) [inline cypress/e2e/dashboard/events/events-index.cy.ts:102]:
```suggestion
    cy.findByRole('link', { name: 'Events' }).click();
    cy.contains('Loading...');

File: cypress/e2e/dashboard/events/events-index.cy.ts [hunk 5]
```diff
@@ -140,7 +139,6 @@ describe('spec needing owner', () => {
     cy.get('button[aria-label="Options"]').click();
     cy.findByRole('menuitem', { name: 'Dashboard' }).click();
     cy.findByRole('link', { name: 'Events' }).click();
```
Comment (Sboonny) [inline cypress/e2e/dashboard/events/events-index.cy.ts:141]:
```suggestion
    cy.findByRole('link', { name: 'Events' }).click();
    cy.contains('Loading...');

**Review by ojeytonwilliams — APPROVED** [review]

**Pull request merged**